### ISSUE TITLE
Add WAMP message internal-only attributes for publisher/caller disclosure

### ISSUE THREAD

**Issue by oberstet** [crossbario/autobahn-python#1392]
from https://crossbario.slack.com/archives/C0473JCQ6/p1591287619078900

would work like this:

* sessions embedded inside the router worker (such as rlinks) are using https://github.com/crossbario/crossbar/blob/7a24af99fce6d057c63a9c7b144b0418ca627f4e/crossbar/router/session.py#L69
* such session received WAMP messages here https://github.com/crossbario/crossbar/blob/master/crossbar/router/session.py#L273
* so the event handler in rlink gets a native python WAMP msg object (without previous serialization/deserialization)
* so we could just stick the publisher session ID/authid/authrole into a non-serialized attribute here https://github.com/crossbario/autobahn-python/blob/master/autobahn/wamp/message.py#L3141 ..
* .. and omit this internal-use-only attribute in https://github.com/crossbario/autobahn-python/blob/effc130cd14ae16a19521129ba290b07d287f845/autobahn/wamp/message.py#L3742

IOW, like:

``` 
Event._internal_only_publisher
Event._internal_only_publisher_authid
Event._internal_only_publisher_authrole
Call._internal_only_caller
Call._internal_only_caller_authid
Call._internal_only_caller_authrole
``` 

we can probably do that generically, for any WAMP message sent/received in Message https://github.com/crossbario/autobahn-python/blob/master/autobahn/wamp/message.py#L362

**Comment by oberstet** [comment]
background:

* rlinks (for router-to-router) need to have the original publisher/caller to fill in `forward_for`
* since rlinks sit as regular, but router-embedded sessions on the originating realm, they only currently get access to this information if the realm was configured to disclose the publisher/caller

above provides the basis to work around this limitation in CB.

**Comment by meejah** [comment]
We could just add `_router_internal` (or similar) to Message -- and thus assign anything at all that the router wants.

**Comment by oberstet** [comment]
^ yeah, agreed! that looks cleaner (only 1 attribute, and having flexibility rgd what we stick to that ..)

**Comment by meejah** [comment]
This is fixed in #1393
